static/app/utils/analytics/feedbackAnalyticsEvents.tsx

```diff
@@ -2,17 +2,13 @@ export type FeedbackEventParameters = {
   'feedback.details-integration-issue-clicked': {
     integration_key: string;
   };
-  'feedback.index-old-ui-clicked': {};
-  'feedback.index-setup-button-clicked': {};
   'feedback.index-setup-viewed': {};
   'feedback.list-item-selected': {};
 };
 
 export type FeedbackEventKey = keyof FeedbackEventParameters;
 
 export const feedbackEventMap: Record<FeedbackEventKey, string | null> = {
-  'feedback.index-setup-button-clicked': 'Clicked Feedback Onboarding Setup Button',
-  'feedback.index-old-ui-clicked': 'Clicked Go To Old User Feedback Button',
```

Comment:
moved these into the button itself with `analyticsEventKey` and `analyticsEventName`